idea/indexer/src/rabbitmq/index.ts

```diff
@@ -100,6 +98,11 @@ export class RMQService {
     this.mainChannel.publish(exchange, queue, Buffer.from(messageBuff), { correlationId });
   }
 
+  static sendMsgToMetaStorage(exchange: RabbitMQExchanges, queue: RabbitMQueues, params: any, method?: string): void {
+    const messageBuff = JSON.stringify(params);
+    mainChannel.publish(exchange, queue, Buffer.from(messageBuff), { headers: { method } });
+  }
+
```

Comment:
Oh, maybe we can do it in some other way? I mean without static method

Reply:
What is the big problem if use this way? Another way to pass a `rabbitMQ` instance is like a constructor parameter in `indexer` and also in `temp-state`.